legend-engine-ide-lsp-extension-api/src/main/java/org/finos/legend/engine/ide/lsp/extension/completion/LegendCompletion.java

```diff
@@ -0,0 +1,72 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.engine.ide.lsp.extension.completion;
+
+import java.util.Objects;
+
+public class LegendCompletion
+{
+    private final String description;
+
+    private final String suggestion;
+
+    public LegendCompletion(String description, String suggestion)
+    {
+        this.description = description;
```

Comment:
description is used on equal and hashcode as if it is not noll too...

Reply:
Agreed, fixed